### DISCUSSION TITLE
Autofill adds double quotes by default in Version 23.1.1.202306251800

### DISCUSSION THREAD

**Discussion by Ilkka-Vaahtoranta** [dbeaver/dbeaver#20398] [Q&A] [answered]
After latest upgrade to Version 23.1.1.202306251800 autofill adds double quotes to all entries and does not recognize objects in case they are not double quoted

example that autofill would have done before
SELECT column1 FROM schema.table;

example that autofill does now
SELECT "column1" FROM "schema"."table";

This is mostly irritating and I cannot find any setting in preferences to turn it of or change the behavior. I'm using Snowflake as the database.

**Comment by E1izabeth** [comment]
The issue is #20391. We're going to fix it in 23.1.2

**Comment by E1izabeth** [comment, marked as answer]
Was fixed in https://github.com/dbeaver/dbeaver/issues/20391
The fix will be available in the next release.
